corehq/apps/locations/static/locations/js/location_tree.async.js

```diff
@@ -73,6 +75,7 @@ hqImport('locations/js/location_tree', function() {
         var model = this;
         this.selected_location = ko.observable();
         this.l__selected_location_id = ko.observable();
+        this.clearLocationSelection = ko.observable(options.clearLocationSelection);
```

Comment:
This probably doesn't need to be an observable (same goes for the other observables added in this commit). Observables are generally for values that are going to be changing frequently/unpredictably (like user inputs), where you want other knockout code to be able to subscribe to changes. It's fine for a knockout model to have some properties that are observables and some that are plain js.

Reply:
Ah that's cool. I didn't know what had to be an observable. I was thinking it might be something that could be "observed", but it's nice to know that plain js can also be read